composer.json

```diff
@@ -7,7 +7,8 @@
         "php": ">=5.4.0",
         "evenement/evenement": "~2.0",
         "react/event-loop": "0.4.*",
-        "react/stream": "0.4.*"
+        "react/stream": "0.4.*",
+		"phpunit/phpunit": "~4.0"
```

Comment:
This shouldn't be necessary, as `sebastian/environment` was already a dev dependency. Even if we were to add PHPUnit as a dependency, it should be included under `require-dev`. That said, I'm of the opinion that it's redundant though, as we can expect developers and Travis to have it already.